.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,226 @@
+            REDIS_HOST: redis
+            CLICKHOUSE_HOST: clickhouse
+            CLICKHOUSE_PORT: 9000
+          ports:
+            - 1218:1218
+
+        redis:
+          image: redis:5.0-alpine
+          ports:
+            - 6379:6379
+
+        memcached:
+          image: memcached:1.5-alpine
+          ports:
+            - 11211:11211
+
+        postgres:
+          image: postgres:9.6
+          env:
+            POSTGRES_USER: postgres
+            POSTGRES_PASSWORD: postgres
+          ports:
+            # Maps tcp port 5432 on service container to the host
+            - 5432:5432
+          # needed because the postgres container does not provide a healthcheck
+          options: >-
+            --health-cmd pg_isready
+            --health-interval 10s
+            --health-timeout 5s
+            --health-retries 15
+
+        zookeeper:
+          image: confluentinc/cp-zookeeper:4.1.0
+          env:
+            ZOOKEEPER_CLIENT_PORT: 2181
+
+        kafka:
+          image: confluentinc/cp-kafka:5.1.2
+          env:
+            KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+            KAFKA_LISTENERS: INTERNAL://0.0.0.0:9093,EXTERNAL://0.0.0.0:9092
+            KAFKA_ADVERTISED_LISTENERS: INTERNAL://kafka:9093,EXTERNAL://kafka:9092
+            KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
+            KAFKA_INTER_BROKER_LISTENER_NAME: INTERNAL
+            KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+
+      steps:
+        # Checkout codebase
+        - uses: actions/checkout@v2
+
+        # See https://github.com/actions/cache/blob/master/examples.md#node---yarn for example
+        - name: Get yarn cache dir
+          id: yarn-cache-dir-path
+          run: echo "::set-output name=dir::$(yarn cache dir)"
+
+        # Use nvmrc so we don't have yet another place node version is defined
+        - name: Get node version from .nvmrc
+          id: nvmrc
+          run: echo "::set-output name=version::$(cat .nvmrc)"
+
+        # yarn cache
+        - uses: actions/cache@v1
+          id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+          with:
+            path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+            restore-keys: |
+              ${{ runner.os }}-yarn-
+
+        # Setup node
+        - uses: actions/setup-node@v1
+          with:
+            node-version: ${{ steps.nvmrc.outputs.version }}
+
+        # Use `.python-version` to avoid duplication
+        - name: Get python version from `.python-version`
+          id: python-version
+          run: echo "::set-output name=version::$(cat .python-version)"
+
+        # setup python
+        - name: Set up Python ${{ steps.python-version.outputs.version }}
+          uses: actions/setup-python@v1
+          with:
+            python-version: ${{ steps.python-version.outputs.version}}
```

Comment:
Ugh @joshuarli 
![image](https://user-images.githubusercontent.com/79684/81014314-7bc02680-8e11-11ea-9402-c725ef5d66c5.png)

Reply:
Yeah, rip. https://github.com/getsentry/sentry/pull/18576#discussion_r419731069

I only found https://github.com/gabrielfalcao/pyenv-action with a quick search, but I definitely don't think it's worth it to test on 2.7.16. Pin to 2.7.17, it's fine.